packages/aws-rfdk/lib/deadline/test/thinkbox-docker-recipes.test.ts

```diff
@@ -129,40 +132,13 @@ describe('ThinkboxDockerRecipes', () => {
     expect(actualImage.sourceHash).toEqual(expectedImage.sourceHash);
   });
 
-  describe('provides the Deadline version with', () => {
-    let version: IVersion;
-    beforeEach(() => {
-      // WHEN
-      const recipes = new ThinkboxDockerRecipes(stack, 'Recipes', {
-        stage,
-      });
-      version = recipes.version;
+  test('provides the Deadline version', () => {
```

Comment:
These changes don't look right. There weren't, and shouldn't be, any changes to the `ThinkboxDockerRecipes` construct, so the test shouldn't change.

Reply:
The version belonging to `ThinkboxDockerRecipes` comes from `Stage` and is an instance of a `VersionQuery` now. Since these values are outputs of the CustomResource in `VersionQuery`, we can't validate them in a unit test. I think I can improve the `expectCDK` statement to validate that the `Custom::RFDK_DEADLINE_INSTALLERS` has `FULL_VERSION_STRING` in its input.